test/unit/pdf.image_decoders_spec.js

```diff
@@ -37,7 +37,7 @@ describe("pdfimage_api", function () {
       getVerbosityLevel,
       Jbig2Image,
       JpegImage,
-      JpxImage,
```

Comment:
Could OpenJPEG also be used to e.g. replace our custom JPEG decoder in `jpg.js` and therefore the `JpegImage` class as well? Obviously that'd not be in scope of this PR, but I wanted to ask because that'd be really nice and also be a factor for determining when we'll do the major bump from the comment above. (I don't think OpenJPEG does JBIG2 but maybe we can also replace that one with a better upstream version later.)

Reply:
I *think* that this decoder is only for JPEG2000, given https://github.com/uclouvain/openjpeg?tab=readme-ov-file#what-is-openjpeg-